resources/testharness.js

```diff
@@ -2788,6 +2920,50 @@ policies and contribution forms [3].
         }
     }
 
+    /**
+     * Immediately invoke a "predicate" function with a series of values in
+     * parallel and invoke a final "done" function when all of the "predicates"
+     * invocations have signaled completion.
+     *
+     * @param {array} value Zero or more values to use in the invocation of
+     *                      `predicate`
+     * @param {function} predicate A function that will be invoked once for
```

Comment:
Calling this a predicate seems a bit weird to me, at least it's not a https://en.wikipedia.org/wiki/Predicate_(mathematical_logic) which is what I usually expect in code. Maybe just "callback", or something else?

Reply:
This is tough because the API expects two callback functions. What do you think
of "iteratee", is in [`_.forEach`](https://lodash.com/docs/#forEach)?

Reply:
That would work for me, I have a soft spot for made up somethingee words. "iterCallback" and "doneCallback" would also WFM.

Reply:
I like those, too. I've used the "snake case" versions in observance of local code style convention.